pkgs/development/interpreters/python/update-python-libraries/update-python-libraries.py

```diff
@@ -274,6 +322,9 @@ def _update_package(path, target):
 
     text = _replace_value('version', new_version, text)
     text = _replace_value('sha256', new_sha256, text)
+    if fetcher == 'fetchFromGitHub' and prefix:
+        _replace_value('rev', f"{prefix}${{version}}", text)
```

Comment:
need to explicitly set version, see link below
```suggestion
    if fetcher == 'fetchFromGitHub':
        if prefix:
            _replace_value('rev', f"{prefix}${{version}}", text)
        else:
            _replace_unquoted_value('rev', "version", text)
```
https://github.com/NixOS/nixpkgs/pull/99636#discussion_r499898459

Reply:
I could also do `rev = "${version}";`, but it would be a lot cleaner to do `rev = version;`

Reply:
came up with a better solution, just re-write `"${version}";` to be `version;` after applying the update